core/tests/test_utils.py

```diff
@@ -71,11 +72,27 @@ def __init__(self, service_name='urlfetch'):
         self.request = None
 
     def set_return_values(self, content='', status_code=200, headers=None):
+        """Set mock content, status_code and headers for Url Service mock
+        object's return_values.
+
+        Args:
+            content: Content for the return_values. Defaults to empty string.
```

Comment:
Add type information. No need to mention defaults. Follow this format:

content: str. The content to return in subsequent calls to the urlfetch mock.